examples/game/alien_cake_addict.rs

```diff
@@ -22,38 +17,26 @@ fn main() {
         .insert_resource(Msaa { samples: 4 })
         .init_resource::<Game>()
         .add_plugins(DefaultPlugins)
-        .insert_resource(State::new(GameState::Playing))
+        .add_state(GameState::Playing)
         .add_startup_system(setup_cameras.system())
-        .add_stage_after(
-            CoreStage::Update,
-            GameStage::InGame,
-            StateStage::<GameState>::default(),
-        )
-        .on_state_enter(GameStage::InGame, GameState::Playing, setup.system())
-        .on_state_update(GameStage::InGame, GameState::Playing, move_player.system())
-        .on_state_update(GameStage::InGame, GameState::Playing, focus_camera.system())
-        .on_state_update(GameStage::InGame, GameState::Playing, rotate_bonus.system())
-        .on_state_update(
-            GameStage::InGame,
-            GameState::Playing,
-            scoreboard_system.system(),
+        .add_system_set(State::on_enter_set(GameState::Playing).with_system(setup.system()))
+        .add_system_set(
+            State::on_update_set(GameState::Playing)
```

Comment:
`on_enter_set` and friends is a bit of a confusing name in English: "set" could either be a verb (to set a property on update...) or a "noun" (a collection of systems). I think it's the latter here from context.

IMO `State::on_enter` (etc.) would be clearer in typical usage, since you're seeing the system set call just above it already.

Reply:
Ah I see, those names are reserved for the transition functions... 

What about `on_enter_systems`?